### PR TITLE
[11.0][MIG] document page procedure

### PR DESCRIPTION
**Pull request by jalzaga**
Module document_page_procedure Migration to version 11.0 #200

### CODE REVIEW

File: document_page_procedure/migrations/7.0.1.0/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+##############################################################################
```
Comment (ivantodorovich) [inline document_page_procedure/migrations/7.0.1.0/pre-migration.py]: remove old migration script

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Minor comment

I'm unsure about the resolution of git conflicts on the last commit. 
Is this the way to go when migrating modules?

**Comment by jalzaga** [comment]
@max3903 @pedrobaeza I cannot see what is the problem with the runbot. Could you please give me a clue?

**Comment by ivantodorovich** [comment]
Seems to be unrelated to this PR. I've seen it failing like this in other PRs aswell

**Review by ivantodorovich — APPROVED** [review]

File: document_page_procedure/README.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+To enter your procedures, you need to:
+
+* go to Management Systems > Documentation > Procedures
+* click on create and select the 'Procedure' category
+* add your content using the provided template
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/118/10.0
```
Comment (max3903) [inline document_page_procedure/README.rst]: /11.0

File: document_page_procedure/README.rst [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+* Report any issue or ideas to https://github.com/OCA/management-system/issues
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/management-system/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/management-system/issues/new?body=module:%20document_page_procedure%0Aversion:%2010.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (max3903) [inline document_page_procedure/README.rst]: 11.0

File: document_page_procedure/README.rst [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
+
+Changelog
```
Comment (max3903) [inline document_page_procedure/README.rst]: This section can be removed.

File: document_page_procedure/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2010 Savoir-faire Linux (<http://www.savoirfairelinux.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Document Management - Wiki - Procedures",
+    "version": "11.0.1.0.0",
+    "author": "Savoir-faire Linux,Odoo Community Association (OCA)",
+    "website": "http://www.savoirfairelinux.com",
```
Comment (max3903) [inline document_page_procedure/__manifest__.py]: Can be replaced with https://github.com/OCA/management-system.

**Review by max3903 — CHANGES_REQUESTED** [review]
Minor fixes. 

It would be nice to split the README, but it can be done in a later PR. It's not blocking for this one.

Thanks!

**Comment by jalzaga** [comment]
Could someone unlock this PR? the error seems not to be related with the PR itself @ivantodorovich @max3903

**Comment by max3903** [comment]
@jalzaga Can you take care of my comments? Please bump the version to 11.0 in the readme.

File: document_page_procedure/README.rst (hunk 3 above)
Comment (max3903) [reply document_page_procedure/README.rst]: `version:%2011.0%0A%0A`

**Comment by jalzaga** [comment]
Done @max3903

**Review by max3903 — APPROVED** [review]

**Pull request merged**